Packages/com.unity.multiplayer.samples.coop/Utilities/SceneManagement/SceneLoaderWrapper.cs

```diff
@@ -126,6 +125,18 @@ void OnSceneEvent(SceneEvent sceneEvent)
                         m_ClientLoadingScreen.StopLoadingScreen();
                     }
                     break;
+                case SceneEventType.Synchronize: // Server told client to start synchronizing scenes
+                {
+                    // Only executes on client that is not the host
+                    if (NetworkManager.IsClient && !NetworkManager.IsHost)
+                    {
+                        // unload all currently loaded additive scenes so that if we connect to a server with the same
+                        // main scene we properly load and synchronize all appropriate scenes without loading a scene
+                        // that is already loaded.
+                        UnloadAdditiveScenes();
```

Comment:
A better solution than this could be made to avoid duplicate scenes when reconnecting once [this issue](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/issues/1874) is solved in NGO

Reply:
Putting on hold until it is

Reply:
After discussion, knowing workaround is pretty small, let's identify which lines are workarounds and waiting for which fix. (+jira ticket on BR side to track this)